lhotse/augmentation.py

```diff
@@ -0,0 +1,153 @@
+            src_info={
+                'channels': audio.shape[0],
+                'length': audio.shape[1],
+                'rate': self.sampling_rate,
+            },
+            target_info={
+                'channels': 1,
+                'length': audio.shape[1],
+                'rate': self.sampling_rate,
+            }
+        )
+
+        # Sox might misbehave sometimes by giving nan/inf if sequences are too short (or silent)
+        # and the effect chain includes eg `pitch`
+        if torch.isnan(augmented).any() or torch.isinf(augmented).any():
+            warnings.warn('NaN/Inf encountered in augmented sox output - returning non-augmented audio.')
+            return audio.clone()
+        return augmented
+
+
+_DEFAULT_AUGMENTATIONS = {}
+
+
+def available_wav_augmentations() -> List[str]:
+    """Return a list of augmentation setups available in Lhotse."""
+    return list(_DEFAULT_AUGMENTATIONS.keys())
+
+
+def register_wav_augmentation(fn):
+    """
+    Register a new augmentation setup that can be retrieved by the name of the function passed in.
+
+    :param fn: a function returning an ``augment.EffectChain`` that performs wav augmentation.
+    """
+    _DEFAULT_AUGMENTATIONS[fn.__name__] = fn
+    return fn
+
+
+@register_wav_augmentation
+def pitch(sampling_rate: int):
+    """
+    Returns a pitch modification effect for wav augmentation.
+
+    :param sampling_rate: a sampling rate value for which the effect will be created (resampling is needed for pitch).
+    """
+
+    import augment
+    effect_chain = augment.EffectChain()
+    # The pitch effect changes the sampling ratio; we have to compensate for that.
+    # Here, we specify 'quick' options on both pitch and rate effects, to speed up things
+    effect_chain.pitch("-q", _random_pitch_shift).rate("-q", sampling_rate)
+    return effect_chain
+
+
+@register_wav_augmentation
+def reverb(*args, **kwargs):
+    """
+    Returns a reverb effect for wav augmentation.
+    """
+
+    import augment
+    effect_chain = augment.EffectChain()
+    # Reverb it makes the signal to have two channels,
+    # which we combine into 1 by running `channels` w/o parameters
+    effect_chain.reverb(50, 50, _random_room_size).channels()
+    return effect_chain
+
+
+@register_wav_augmentation
+def pitch_reverb_tdrop(sampling_rate: int):
+    """
+    Returns an effect chain composed of pitch modification, reverberation and time dropout proposed in:
+    https://github.com/facebookresearch/WavAugment/blob/master/examples/python/librispeech_selfsupervised.py#L152
+    https://arxiv.org/abs/2007.00991
+
+    :param sampling_rate: a sampling rate value for which the effect will be created (resampling is needed for pitch).
+    """
+
+    import augment
+    effect_chain = augment.EffectChain()
+    # The pitch effect changes the sampling ratio; we have to compensate for that.
+    # Here, we specify 'quick' options on both pitch and rate effects, to speed up things
```

Comment:
would be nice to make clear the meaning of these params, e.g. what is the -300:300?